src/st_stuff.c

```diff
@@ -499,6 +502,31 @@ static boolean CheckConditions(sbarcondition_t *conditions, player_t *player)
                 result &= (!!cond->param == false);
                 break;
 
+            // Woof! additions
+            case sbc_healthgreaterequal:
+                result &= player->health >= cond->param;
+                break;
+
+            case sbc_ammogreaterequal_percent:
+                {
+                    ammotype_t type = weaponinfo[player->readyweapon].ammo;
+                    if (type != am_noammo)
```

Comment:
This also needs a check for `player->maxammo[type] > 0`. Also, I am missing the ability to turn ammo blue if it is more than the non-backpack max-value.

Reply:
Fixed in current version.